composer.json

```diff
@@ -59,8 +59,8 @@
         "symfony/process": "3.4.2"
     },
     "require-dev": {
-        "helmich/phpunit-json-assert": "^2.1 || ^3.0",
-        "phpunit/phpunit": "^8"
+        "helmich/phpunit-json-assert": "^3.0",
```

Comment:
[3.x is the only version that supports PHPUnit 8](https://github.com/martin-helmich/phpunit-json-assert#compatibility); we can't use 2.x in any case